testsuite/classlibrary/TestPmono.sc

```diff
@@ -0,0 +1,45 @@
+
+TestPmono : UnitTest {
+	var server;
+
+	setUp {
+		server = Server(this.class.name);
```

Comment:
is a server actually needed for these tests? they seem to pass without it

Reply:
Hm, could do without the server, yeah. I'll take that out.